packages/freezed/lib/src/freezed_generator.dart

```diff
@@ -438,8 +482,8 @@ Read here: https://github.com/rrousselGit/freezed/blob/master/packages/freezed/C
   }
 
   Iterable<CloneableProperty> _commonCloneableProperties(
-    List<ConstructorDetails> constructors,
-    List<Property> commonProperties,
+    Iterable<ConstructorDetails> constructors,
```

Comment:
There are a couple of Iterable changes still in this PR

Reply:
I left these in because when used as a parameter there is no lazy iterable implication but giving more flexibility in the inputs to the function.